.circleci/config.yml

```diff
@@ -20,25 +26,31 @@ jobs:
       - image: cimg/openjdk:8.0.345-node
     environment:
       JVM_OPTS: -Xmx3200m
+    resource_class: large
+    parallelism: 3
```

Comment:
What does this setting `3` do?

Reply:
Parallelism:3 --> It creates 3 different containers to run the IT in parallel